server/src/test/java/io/crate/analyze/SubscriptValidatorTest.java

```diff
@@ -31,94 +31,93 @@
 import io.crate.sql.tree.ArrayLiteral;
 import io.crate.sql.tree.Cast;
 import io.crate.sql.tree.Expression;
-import io.crate.sql.tree.SubscriptExpression;
+import io.crate.sql.tree.IntegerLiteral;
 import io.crate.sql.tree.TryCast;
 
 
 public class SubscriptValidatorTest extends ESTestCase {
 
     private SubscriptContext analyzeSubscript(String expressionString) {
-        SubscriptContext context = new SubscriptContext();
         Expression expression = SqlParser.createExpression(expressionString);
-        SubscriptValidator.validate((SubscriptExpression) expression, context);
-        return context;
+        return SubscriptVisitor.visit(expression);
     }
 
     @Test
-    public void testVisitSubscriptExpression() throws Exception {
+    public void testVisitSubscriptExpression() {
```

Comment:
In general it's fine to keep the throws clauses in tests as there's no harm and it can result in unnecessary churn when making changes. But no need to revert.

If this is coming from a intellij warning, maybe we should check-in some config that disables it.